data/tabular/nomad_structure/meta.yaml

```diff
@@ -0,0 +1,126 @@
+---
+name: nomad-structure
+description: |-
+    A subset from NOMAD dataset, which is a database of DFT computed results of materials.
+    This subset consists of cif structures of around 0.5 million bulk stable materials and their geometric and structural information.
+    All materials in this dataset are modeled using Density Functional Theory using GGA functional.
+targets:
+    - id: density
+      description: Density of the material
+      units: Kg/m^3
+      type: continuous
+      names:
+          - noun: density of the material
+          - noun: density
+      uris:
+    - id: crystal_ststem
+      description: Geometric arrangement of atoms within a crystal
+      type: categorical
+      names:
+          - noun: crystal structure type
+          - noun: geometric arrangement of atoms within a crystal
+          - noun: arrangement of lattice points in space
+      uris:
+    - id: spacegroup
+      description: Spacegroup of the material
+      type: categorical
+      names:
+          - noun: spacegroup of the material
+          - noun: spacegroup
+      uris:
+    - id: pointgroup
+      description: Pointgroup of the material
+      type: categorical
+      names:
+          - noun: pointgroup of the material
+          - noun: pointgroup
+      uris:
+    - id: spacegroup_number
+      description: Spacegroup number of the material
+      type: categorical
+      names:
+          - noun: spacegroup number of the material
+          - noun: spacegroup number
+          - noun: number of the spacegroup in the International Tables for Crystallography
+      uris:
+identifiers:
+    - id: cif
+      type: cif
+      description: CIF
+    - id: formula
+      type: COMPOSITION
+      description: reduced formula
+license: CC BY 4.0
+num_points: 530088
+links:
+    - url: https://nomad-lab.eu/nomad-lab/
+      description: original data source
+bibtex:
+    - |-
+      @article{scheidgen2023nomad,
+      title={NOMAD: A distributed web-based platform for managing materials science research data},
+      author={Scheidgen, Markus and Himanen, Lauri and Ladines, Alvin Noe and Sikter, David and Nakhaee, Mohammad and Fekete, {\'A}d{\'a}m and Chang, Theodore and Golparvar, Amir and M{\'a}rquez, Jos{\'e} A and Brockhauser, Sandor and others},
+      journal={Journal of Open Source Software},
+      volume={8},
+      number={90},
+      pages={5388},
+      year={2023}
+      }
+templates:
```

Comment:
@n0w0f I'm very happy to see that you spend a lot of effort coming up with many different templates